predicators/approaches/bridge_policy_approach.py

```diff
@@ -365,10 +366,79 @@ def __init__(self,
                          action_space, train_tasks, task_planning_heuristic,
                          max_skeletons_optimized)
         self._maple_initialized = False
+        if task_planning_heuristic == "default":
+            task_planning_heuristic = CFG.sesame_task_planning_heuristic
+        self._task_planning_heuristic = task_planning_heuristic
         self._trajs: List[LowLevelTrajectory] = []
+        self.CanPlan = Predicate("CanPlan", [], self._Can_plan)
+        self.CallPlanner = utils.SingletonParameterizedOption(
+            "CallPlanner",
+            types=None,
+            policy=self.call_planner_policy,
+            params_space=Box(0, len(train_tasks), (1, )),
+        )
+        initial_options.add(self.CallPlanner)
+        self._initial_options = initial_options
         self.mapleq=MapleQApproach(self._get_current_predicates(), \
                                    self._initial_options, self._types, \
                                     self._action_space, self._train_tasks)
+        self._current_control = ""
```

Comment:
instead of an empty string, set this to None and then set the type of this variable to `Optional[str]`. In general, it's a good design pattern to use `None` when you don't have a value for a variable; makes errors easier to spot

Reply:
when I set it to None I get an error from the type checker, should I just set it to "planner"?